CMakeLists.txt

```diff
@@ -40,14 +40,8 @@ if(BUILD_TESTS)
 
   enable_testing()
 
-  add_test(CIPIC_subject_003_hrir_final ../tests/compare.sh ../tests/CIPIC_subject_003_hrir_final)
```

Comment:
Please do not remove the test cases.
Instead, please add a file with M,R delays.